tfgrid3/ubuntu24.04/fullvm/ubuntu24-fullvm-flist-builder.sh

```diff
@@ -68,4 +68,4 @@ if [ -z "$API_KEY" ]; then
 	        exit 2
 fi
 
-curl -X POST -H "Authorization: Bearer $API_KEY" -F "file=@ubuntu-jammy.tar.gz" https://hub.grid.tf/api/flist/me/upload
\ No newline at end of file
+curl -X POST -H "Authorization: Bearer $API_KEY" -F "file=@ubuntu-noble.tar.gz" https://hub.grid.tf/api/flist/me/upload
```

Comment:
could you please use the version number in the name of the flist better than using image codename to be more clear for users when adding it to hub then dashboard

Reply:
updated